site-scrapers/index.js

```diff
@@ -1,24 +1,21 @@
-const FamilyPracticeGroup = require("./FamilyPracticeGroup.js");
-const MAImmunizations = require("./MAImmunizations.js");
-const UMassAmherst = require("./UMassAmherst.js");
-const Hannaford = require("./Hannaford.js");
-const Harrington = require("./Harrington.js");
-const Curative = require("./Curative.js");
-const Atrius = require("./Atrius.js");
-const LynnTech = require("./LynnTech.js");
-const PriceChopper = require("./PriceChopper.js");
 
-let scrapers = [
-    FamilyPracticeGroup,
-    MAImmunizations,
-    UMassAmherst,
-    Hannaford,
-    Harrington,
-    Curative,
-    Atrius,
-    LynnTech,
-    PriceChopper
-];
+let scrapers = [];
+
+if(process.argv.length > 2){
```

Comment:
I'm good with this idea! I'm noticing that your spacing isn't following `prettier` standards. Do you have the extension installed? Could you please run it on your changes and check those in?

Reply:
Yeah, there should be away to add arguments to your `node main.js` command in AWS. For now since all sites run at the same time you can just add them all as args. At some point it will probably make sense to run one at a time.

Reply:
I'm not going to change the command to run the script every time we add a new scraper. It's introducing an extra point of error that is extremely manual. Not worth the tradeoff when the alternative is having some commas that cause merge conflicts.